x-pack/functionbeat/function/provider/provider_test.go

```diff
@@ -47,7 +46,7 @@ func TestRunnable(t *testing.T) {
 		}
 
 		errReceived := runnable.Run(context.Background(), telemetry.Ignored())
-		assert.Equal(t, err, e.Cause(errReceived))
+		assert.Equal(t, "could not create a client for the function: "+err.Error(), errReceived.Error())
```

Comment:
special attention to this file

Reply:
If the tests still pass you're good, also this is functionbeat which is deprecated now.